mpf/config_players/blinkenlight_player.py

```diff
@@ -0,0 +1,61 @@
+"""Light config player."""
+from copy import deepcopy
+from mpf.config_players.device_config_player import DeviceConfigPlayer
+from mpf.core.rgb_color import RGBColor
+from mpf.core.utility_functions import Util
+
+
+class BlinkenlightPlayer(DeviceConfigPlayer):
+
+    """Blinks lights different colors based on config."""
+
+    config_file_section = 'blinkenlight_player'
+    show_section = 'blinkenlights'
+    machine_collection_name = 'blinkenlights'
+    allow_placeholders_in_keys = True
+
+    def __init__(self, machine):
+        """Initialise blinklight_player."""
+
+        super().__init__(machine)
+
+    def play(self, settings, context, calling_context, priority=0, **kwargs):
+        """Adds/removes colors to blink."""
+        del kwargs
+
+        for blinkenlight, s in settings.items():
+            action = s['action']
+            if action == 'add':
+                self._add_color(blinkenlight, s['color'], s['key'])
+            elif action == 'remove':
+                self._remove_color(blinkenlight, s['key'])
+            elif action == 'removeall':
+                self._remove_all_colors(blinkenlight)
+            blinkenlight.num_colors = len(blinkenlight.colors)
+            blinkenlight._restart()
```

Comment:
I think this is ok, since now the blinkenlight figures out where it should be in the cycle, so it shouldn't interfere with the sync or cause flickering.